d2l-table-style.html

```diff
@@ -62,24 +62,17 @@
 				@apply(--d2l-table-body);
 			}
 
-			:host > ::content > * > tr,
-			:host > ::content > tr {
+			:host > ::content > * > tr {
 				@apply(--d2l-table-row);
 			}
 
-			:host > ::content > thead > tr > th,
 			:host > ::content > * > tr > td,
-			:host > ::content > tr > td,
-			:host > ::content > tbody > tr > th,
-			:host > ::content > tfoot > tr > th {
+			:host > ::content > * > tr > th {
 				@apply(--d2l-table-cell);
 			}
 
-			:host-context([dir="rtl"]) > ::content > thead > tr > th,
 			:host-context([dir="rtl"]) > ::content > * > tr > td,
-			:host-context([dir="rtl"]) > ::content > tr > td,
-			:host-context([dir="rtl"]) > ::content > tbody > tr > th,
-			:host-context([dir="rtl"]) > ::content > tfoot > tr > th {
+			:host-context([dir="rtl"]) > ::content > * > tr > th {
```

Comment:
The performance of the selector depends mostly on the primary selector. Changing the `*` to `tbody`, `thead`, `tfoot` would increase the number of selectors by 28, and that would negatively affect performance.

I'm assuming that having `> * >` selectors is better than more selectors.